packages/ruby/lib/readme/metrics.rb

```diff
@@ -8,9 +8,12 @@ class Metrics
     SDK_NAME = "Readme.io Ruby SDK"
     ENDPOINT = "https://metrics.readme.io/v1/request"
 
-    def initialize(app, api_key, &get_user_info)
+    def initialize(app, options, &get_user_info)
+      raise("Missing development flag") if options[:development].nil?
```

Comment:
Should the development flag have a default instead of raising?

Reply:
Good question. I think it makes sense to ensure things are configured explicitly before running, but I don't feel strongly about it. If we used one, what would be a good default value here?

Reply:
I think `false` ? What does the node package do?

Reply:
Yeah false makes sense here. That's what the node package does, so I'll update this to match that behavior.